desktop/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin
+/bin
```

Comment:
Again unsure of why this is happening or what it will affect. Should I just delete them both?